.markdown-link-check.json

```diff
@@ -1,5 +1,9 @@
 {
-  "aliveStatusCodes": [200, 206, 403],
+  "aliveStatusCodes": [
+    200,
+    206,
+    403
+  ],
```

Comment:
Changed.

Reply:
Ah, this was because of `clang-format`.
https://github.com/autowarefoundation/autoware.universe/pull/355#issuecomment-1051565661